host/integration_test.go

```diff
@@ -1018,8 +1018,10 @@ func (s *integrationSuite) TestCronWorkflow() {
 		executionInfo := closedExecutions[i]
 		// Roundup to compare on the precision of seconds
 		expectedBackoff := executionInfo.GetExecutionTime()/1000000000 - lastExecution.GetExecutionTime()/1000000000
+		// We are using different time source and the close time and start time can be different
+		executionTimeDiff := executionInfo.GetStartTime()/1000000000 - lastExecution.GetCloseTime()/1000000000
```

Comment:
plz 
1. add a comment here explaining how the execution time is calaulated
2. cut an issue to remove this patch once unified time source is implemented

Reply:
updated